### PR TITLE
jupyter: fixed previewer for jpynb

### PR DESCRIPTION
**Pull request by alejandromumo**
closes https://github.com/inveniosoftware/invenio-app-rdm/issues/1751

EDIT: it seems that tests were also failing on master previously https://github.com/inveniosoftware/invenio-previewer/runs/5771027111?check_suite_focus=true

### CODE REVIEW

File: invenio_previewer/extensions/ipynb.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def render(file):
     notebook = nbformat.reads(content.decode("utf-8"), as_version=4)
 
     html_exporter = HTMLExporter()
-    html_exporter.template_file = "basic"
+    html_exporter.template_file = "base"
```
Comment (slint) [inline invenio_previewer/extensions/ipynb.py:31]: In the docs, they seem to be specifying the `classic` and `lab` templates as well, which generate also inline CSS and use a CDN for loading some JS... I see also [this comment](https://github.com/jupyter/nbconvert/pull/1431#issuecomment-725512612), wondering if that's the expected behaviour.

We can shelve this as a separate and explore if other templates might be better for our use-case.

**Review by slint — APPROVED** [review]
LGTM, one comment on the template selection.

Another concern is if we should bump the minimum version of `nbconvert` (from the current 6.0) to avoid hitting this issue, but it seems like the `base` template was also present in 6.0 (otherwise the `min` tests would fail I guess).

File: invenio_previewer/extensions/ipynb.py (hunk 1 above)
Comment (alejandromumo) [reply invenio_previewer/extensions/ipynb.py:31]: Regarding bump: `base` appeared as a renaming of `basic` [here](https://github.com/jupyter/nbconvert/commit/f4c7daefe0e24055a92f185d6fd803abf909db2f) and this was released on `6.0.0a`. 

Regarding other templates: I agree that we should investigate it as an improvement, I created a [card](https://github.com/orgs/inveniosoftware/projects/97#card-83365296) for that.

**Comment by luweizheng** [comment]
Hi @alejandromumo I have the same issue in invenio-rdm 9.1. Just want to know when to release the newer version of invenio-previewer? Now the latest v1.3.6 does not have this fix.

**Comment by alejandromumo** [comment]
> Hi @alejandromumo I have the same issue in invenio-rdm 9.1. Just want to know when to release the newer version of invenio-previewer? Now the latest v1.3.6 does not have this fix.

Hello there, I am sorry for the delay. 
I created an [issue](https://github.com/inveniosoftware/invenio-previewer/issues/161) so we will try to release it as soon as possible.

**Pull request merged**